pages/common/vagrant.md

```diff
@@ -0,0 +1,19 @@
+# vagrant
+
+> Configure lightweight, reproducible, and portable development environments
+
+- Create Vagrantfile in current folder with the given Vagrant box
+
+`vagrant init hashicorp/precise32`
```

Comment:
I think, it is different example.

When you write something like
{{/path/to/file}}
newbie user has everything he needs, he has command and he has file to process.

In this particular case, if I generalize it with {{box_name}} or anything like this, newbie user will have to go and find what it means. So they will have to go and read yet another documentation. Default Vagrant machine (hashicorp/precise32) covers 75% of all cases.